lib/accessibility-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const Eval = require('./eval')
+const AccessibilityRuleView = require('./accessibility-rule-view')
+const View = require('./view')
+const accessibility = require('./accessibility')
+
+class AccessibilityView extends View {
```

Comment:
Does this filter box work for this view? It looks like it might not be wired up currently.